lms/templates/ux/reference/teams-base.html

```diff
@@ -155,11 +155,218 @@ <h1 class="page-title">Solar Power</h1>
     <div class="page-content">
       <nav class="page-content-nav">
         <a href="" class="nav-item is-active">My Teams</a>
-        <a href="" class="nav-item">Browse All Teams</a>
+        <a href="" class="nav-item">Browse</a>
       </nav>
 
       <div class="page-content-main">
-        <p>Blah blah blah. Go Team!</p>
+
+        <div class="search-tools">
+          <span class="search-count">Viewing 12 of 24 topics</span>
+          <span class="search-sort">sorted by
+          <select name="search-sort">
```

Comment:
Mind syncing this markup with some of the conventions we've added in the edX PL forms work. Here's an example of the select menus in our documentation site:

```html
<div class="field">
    <label class="field-label" for="select_01">Choose your favorite color of the rainbow (default select menu)</label>
    <select class="field-input input-select" id="select_01" name="select_01">
        <option value="">- Choose one -</option>
        <option value="Red">Red</option>
        <option value="Orange">Orange</option>
        <option value="Yellow">Yellow</option>
        <option value="Green">Green</option>
        <option value="Blue">Blue</option>
        <option value="Indigo">Indigo</option>
        <option value="Violet">Violet</option>
    </select>
</div>
```